apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -167,6 +173,21 @@ export const RegisterAccount1 = () => {
     ...useLastLoginInfo(),
   })
 
+  if (isSignedIn) {
+    return (
+      <StyledOuterContainer className="RegisterAccount1">
+        <Box mx={'auto'} mt={15} width={'fit-content'}>
+          <RegisterPageLogoutPrompt
+            onLogout={() => {
+              sessionContext.refreshSession()
```

Comment:
`refreshSession` will re-create the context, causing the component to re-render with an empty access token. Once it re-renders, the registration page is shown.